health-check.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Script to check health of API. It sends a curl command to the health endpoint and checks
+# that the status code is 200.
+# Note: This script can only be used when the application is running inside a container.
+
+set -e
+
+echo "API Health Check..."
+
+CONTAINER_NAME="trusted-intermediary-router-1"
```

Comment:
I don't think this variable is being used.

Reply:
You are right, this was part of the whole checking for the container; great catch!